6212 Иванов Максим Алексеевич Лаб. 2 Вар. 26

Gwyn1300 opened the pull request:
**ФИО:** Иванов Максим
**Номер группы:** 6212  
**Номер лабораторной:** 2  
**Номер варианта:** 26  
**Краткое описание решаемой задачи:** Согласно варианту 26 лабораторной 2 мне надо было скачать музыку, исполненную на трубе, укулеле и арфе, количество композиций, исполненных на каждом инструменте одинаковое.  
**Краткое описание работы кода:** Модуль `FilePathIterator.py` - итератор. Модуль `main.py` является точкой входа в приложение. Модуль `function.py` - набор функций, необходимых для работы. Модуль `Parsing.py` скачивает аудиофайлы и составляет аннотацию.

Gwyn1300 committed:
Начало работы над лабой 2

Gwyn1300 committed:
Добавлено скачивание файлов
Добавлено создание .csv файла

Gwyn1300 committed:
Добавление итератора
Разделение кода на функции

Gwyn1300 committed:
Выполнено

Gwyn1300 committed:
Merge branch 'main' into lab2

Gwyn1300 committed:
Исправил

IlyaOv reviewed (changes requested):
Общие замечания:
1. Путь к папке для сохранения и путь к файлу аннотации необходимо передавать через аргументы командной строки.
2. Код нужно оформить по по PEP8 (порядок импортов, длина строк, количество пустых строк между функций верхнего уровня и т.д.)

Gwyn1300 committed:
Все файлы:
Отформатированы по PEP8
Добавлен аргумент командной строки: Путь к папке сохранения; Путь к файлу аннотации

Gwyn1300 committed:
Форматирование исправлено

Gwyn1300 committed:
Выполненое задание

IlyaOv reviewed (approved):
Зачтено
